### PR TITLE
[k6 tests] Fix broken perf tests - updated page titles

### LINKED ISSUE #51178: Fix broken K6 tests - page titles updated
With https://github.com/woocommerce/woocommerce/pull/50900 some page titles have been updated and that broke some K6 tests.

### PR DESCRIPTION
**Pull request by adimoldovan**
Closes https://github.com/woocommerce/woocommerce/issues/51178

Page titles changed with https://github.com/woocommerce/woocommerce/pull/50900 and this broke some of the K6 perf tests.

- Updated the expected page titles to fix broken tests
- Added a new STORE_NAME constant in config and used it in tests to make it easier to maintain
- Similar, added a FOOTER_TEXT constant
- Added a checkResponse util function and used it in more places where similar checks were performed
- Fixed eslint errors, mostly no-shadow rule violations and formatting

### How to test the changes in this Pull Request:

Ran in CI [here](https://github.com/woocommerce/woocommerce/actions/runs/10722784433/job/29734589107?pr=51173) and it passed.

### CODE REVIEW

File: plugins/woocommerce/package.json [hunk 1]
```diff
@@ -209,7 +209,7 @@
 						"start": "env:test"
 					},
 					"events": [
-						"pull_request",
+						"temp_skip_pull_request",
```
Comment (adimoldovan) [inline plugins/woocommerce/package.json]: Revert before merging

```suggestion
						"pull_request",
```

File: plugins/woocommerce/package.json [hunk 2]
```diff
@@ -542,7 +543,7 @@
 						"start": "env:start:blocks"
 					},
 					"events": [
-						"pull_request",
+						"temp_skip_pull_request",
```
Comment (adimoldovan) [inline plugins/woocommerce/package.json]: Revert before merging

```suggestion
						"pull_request",
```

File: plugins/woocommerce/package.json [hunk 3]
```diff
@@ -490,6 +490,7 @@
 						"start": "env:perf"
 					},
 					"events": [
+						"pull_request",
```
Comment (adimoldovan) [inline plugins/woocommerce/package.json]: Revert before merging

```suggestion
```

File: plugins/woocommerce/tests/performance/utils.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+		[ `title is: "${ page.title }"` ]: ( r ) =>
+			r.html().find( 'head title' ).text() === page.title,
+		[ `body contains: ${ page.body }` ]: ( r ) =>
+			r.body.includes( page.body ),
+		[ `footer contains: ${ page.footer }` ]: ( r ) =>
+			r.html().find( 'body footer' ).text().includes( page.footer ),
+	} );
+};
+
+export { checkResponse };
```
Comment (lanej0) [inline plugins/woocommerce/tests/performance/utils.js:30]: Great improvement abstracting this.  Makes the tests much easier to maintain.

**Review by lanej0 — APPROVED** [review]
Solid improvements, and fixes the broken titles. 👍🏼

**Pull request merged**